src/main/kotlin/pt/up/fe/ni/website/backend/dto/auth/PassRecoveryDto.kt

```diff
@@ -0,0 +1,5 @@
+package pt.up.fe.ni.website.backend.dto.auth
+
+data class PassRecoveryDto(
+    val password: String
```

Comment:
I'd consider putting a constraint to enforce a minimum size to the password, as we have in the account model. Currently, I can reset to an easy password like `1`

Reply:
While doing this, I noticed that `ChangePasswordDto` has the same problem. Should I fix it in this PR?

Reply:
Yes that would be great!